src/com/yetanalytics/lrs_admin_ui/db.cljs

```diff
@@ -80,6 +80,57 @@
 (s/def ::oidc-auth boolean?)
 (s/def ::oidc-enable-local-admin boolean?)
 
+(s/def ::enable-admin-status boolean?)
+
+(s/def :status.data/statement-count nat-int?)
+(s/def :status.data/actor-count nat-int?)
+(s/def :status.data/last-statement-stored (s/nilable string?))
+;; The JSON->edn conversion makes the platform a keyword, so we handle that in
+;; the sub.
+(s/def :status.data/platform-frequency (s/map-of keyword? nat-int?))
```

Comment:
Wouldn't it be better to convert the platform name into a string in the query response handler, so that it's stored as a string?

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/b50d3dd2f3d2848d036068c841d47f5c6f4b438a

Reply:
Ok you can remove the comment now.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/9846fe86c9b1a7a01ff63d873380eb61e8f08ede